### ISSUE TITLE
Chromium pps crashing after starting privileged coroot-node-agent contaiener on Ubuntu 24.10

### ISSUE THREAD

**Issue by devoli170** [coroot/coroot#363]
Hi,

On my first attempt to install the coroot container stack I ran into an issue on my ubuntu 24.10.

After starting the containers with [docker compose ](https://github.com/coroot/coroot/blob/main/deploy/docker-compose.yaml) all my browser tabs crashed with a SEGV and my VSCode crashed aswell.

After stopping the container stack I was able to restart/reload my chromium based apps.

After that, i tested the swarm stack. 
`docker stack deploy -c - coroot`  worked fine, but after starting the privileged `coroot-node-agent` container my apps crashed again. See this screenshot:

![vscode_crash](https://github.com/user-attachments/assets/07948d2f-3fbb-4ace-9171-233fedac077a)

Please let me know if you need more information.

Kind regards

**Comment by devoli170** [comment]
Same issue when running the service on the host with the install script from [here](https://raw.githubusercontent.com/coroot/coroot-node-agent/main/install.sh)

Here some more information about my system:

```
lscpu
Architecture:             x86_64
  CPU op-mode(s):         32-bit, 64-bit
  Address sizes:          48 bits physical, 48 bits virtual
  Byte Order:             Little Endian
```

and kernel
```
uname -r
6.11.0-9-generic
```

**Comment by def** [comment]
Exit code 139 (SIGSEGV) means the kernel stopped your app because it tried to access memory that doesn't exist or is restricted. It's very unlikely Coroot's agent caused this, as it only observes and doesn't modify anything

**Comment by T100D** [comment]
Experiencing the same issue with ubuntu based system and coroot directly installed on it.

While installing and running coroot / coroot-node-agent / coroot-cluster-agent no problem wit chrome or firefox, after the reboot the same error on chrome and firefox (Error code: SIGSEGV)

Linux pop-os 6.12.10-76061203-generic #202412060638~1742504666~22.04~3d20e1a SMP PREEMPT_DYNAMIC Thu M x86_64 x86_64 x86_64 GNU/Linux

**Comment by devoli170** [comment]
@T100D no, I didn't investigate this further. I downgraded to 24.04.1-Ubuntu in the meantime and  did a quick test according to https://docs.coroot.com/installation/ubuntu/. The issue occurs there aswell

**Comment by T100D** [comment]
Installed the latest node-agent (1.23.16) , problem still exists. Starting Chrome wit --no-sandbox helps me out so far.

We are not having this on Rocky linux.
